workflows/ingest/masters.go

```diff
@@ -55,91 +54,42 @@ func Masters(ctx workflow.Context, params MasterParams) (*MasterResult, error) {
 		return nil, err
 	}
 
-	if utils.IsMedia(result.Path.Local()) {
-		// This isn't run on VB masters in old system, but see no reason to not run it here.
-		result.AnalyzeResult, err = analyzeAudioAndSetMetadata(ctx, result.AssetID, result.Path)
```

Comment:
did we lose this normalization now or is it happening somewhere else?

Reply:
This is not normalization, it's just analysis, and noone ever read that data in MB